tools/image_processing/imagej2/imagej2_analyze_skeleton/imagej2_analyze_skeleton.xml

```diff
@@ -78,18 +78,28 @@
         </test>
     </tests>
     <help>
+
+.. class:: warningmark
+
+The underlying AnalyzeSkeleton Fiji plugin works on binary images, so other image types will
+automatically be converted to binary before they are analyzed.  This step is performed using the
+ImageJ2 **Make Binary** command with the following settings: **Iterations:** 1, **Count:** 1,
+**Black background:** No, **Pad edges when eroding:** No.  If these settings are not appropriate,
+first manually convert the image to binary using the **Convert to binary (black and white) with
```

Comment:
@bgruening @blankenberg There are 2 tools in the Galaxy core, both of which I need: ~/tools/meme/meme.xml and ~/tools/meme/fimo.xml.  However, it looks like only the fimo.xml tool has been included in the tools-devteam repo.  I'm thinking the best approach will be to move fimo.xml out of tools-devteam and into tools-iuc, and add the meme.xml to tools-iuc so that they are both located in the same repo.